l5kit/l5kit/environment/envs/l5_env.py

```diff
@@ -0,0 +1,185 @@
+import random
+from collections import defaultdict
+from typing import Any, DefaultDict, Dict, List, NamedTuple
+
+import gym
+import numpy as np
+import torch
+from gym import spaces
+
+from l5kit.configs import load_config_data
+from l5kit.data import ChunkedDataset, LocalDataManager
+from l5kit.dataset import EgoDataset
+from l5kit.environment.cle_metricset import SimulationOutputGym
+from l5kit.environment.reward import Reward, RewardInput
+from l5kit.environment.utils import convert_to_dict, default_collate_numpy, rescale_action
+from l5kit.rasterization import build_rasterizer
+from l5kit.simulation.dataset import SimulationConfig, SimulationDataset
+from l5kit.simulation.unroll import ClosedLoopSimulator, UnrollInputOutput
+
+
+class GymStepOutput(NamedTuple):
+    """ The output dict of gym env.step
+
+    :param obs: the next observation on performing environment step
+    :param reward: the reward of the current step
+    :param done: flag to indicate end of episode
+    :param info: additional information
+    """
+
+    obs: Dict[str, np.ndarray]
+    reward: float
+    done: bool
+    info: Dict[str, Any]
+
+
+class L5Env(gym.Env):
+    """ Custom Environment of L5 Kit that can be registered in OpenAI Gym. """
+
+    def __init__(self, env_config_path: str, sim_cfg: SimulationConfig, reward: Reward, cle: bool) -> None:
+        """
+        :param env_config_path: path to the L5Kit environment configuration file
+        :param simulation_cfg: configuration of the L5Kit closed loop simulator
+        :param reward: calculates the reward for the gym environment
+        :param cle: flag to enable close loop environment updates
+        """
```

Comment:
This docstring should to together with the one above (Custom Environment of ...)